examples/adaptive_library/DXFImport.py

```diff
@@ -0,0 +1,205 @@
+#    **Scripting -> Settings** and select 'Update to Latest Release' under PyMotorCAD updates.
+
+# %%
+# This script is designed to be run from Motor-CAD template "e8". The "e8" template file contains an
+# imported custom geometry from a .dxf file. The template should be adjusted slightly to more
+# closely match the imported custom geometry before running the Adaptive Templates script.
+# The following Standard Template geometry changes should be made:
+#
+# * Set L1 Mag Gap Inner and L2 Mag Gap Inner to 0 mm (for Magnet Layers 1 and 2).
+#
+# * Set L2 Web Thickness to 18 mm (for Magnet Layer 2).
+#
+# * Set L2 Pole Arc [ED] to 105 electrical degrees (for Magnet Layer 2).
+#
+# * Set L2 RDuct Rad Dia to 123.8 mm (for Rotor Duct Layer 2).
+#
+# * Set L2 RDuct Dia to 4.94 mm (for Rotor Duct Layer 2).
+
+# %%
+# .. image:: ../../images/DXFImport1_2.png
+
+# %%
+# If no Motor-CAD file is open, the e8 template will be loaded and the geometry will be adjusted as
+# described above.
+
+# %%
+# Perform required imports
+# ------------------------
+# Import ``pymotorcad`` to access Motor-CAD.
+# Import ``os``, ``shutil``, ``sys``, and ``tempfile``
+# to open and save a temporary .mot file if none is open.
+
+# sphinx_gallery_thumbnail_path = 'images/Adaptive_DXF_Import_5.png'
+import os
+import shutil
+import sys
+import tempfile
+
+import ansys.motorcad.core as pymotorcad
+
+# %%
+# Connect to Motor-CAD
+# --------------------
+# If this script is loaded into the Adaptive Templates file in Motor-CAD, the current Motor-CAD
+# instance is used.
+#
+# If the script is run externally, these actions occur: a new Motor-CAD instance is opened,
+# the e8 IPM motor template is loaded, the geometry changes described above are applied and the file
+# is saved to a temporary folder.
+# To keep a new Motor-CAD instance open after executing the script, use the
+# ``MotorCAD(keep_instance_open=True)`` option when opening the new instance.
+# Alternatively, use the ``MotorCAD()`` method, which closes the Motor-CAD instance after the
+# script is executed.
+
+if pymotorcad.is_running_in_internal_scripting():
+    # Use existing Motor-CAD instance if possible
+    mc = pymotorcad.MotorCAD(open_new_instance=False)
+else:
+    mc = pymotorcad.MotorCAD(keep_instance_open=True)
+    # Disable popup messages
+    mc.set_variable("MessageDisplayState", 2)
+    mc.set_visible(True)
+    mc.load_template("e8")
+
+    # Set Standard Template geometry to closely match the imported .dxf custom geometry
+    mc.set_array_variable(
+        "VShape_Magnet_ClearanceInner", 0, 0
+    )  # Set the Magnet Inner Gap to 0 mm for Layer 1
+    mc.set_array_variable(
+        "VShape_Magnet_ClearanceInner", 1, 0
+    )  # Set the Magnet Inner Gap to 0 mm for Layer 2
+    mc.set_array_variable("WebThickness_Array", 1, 18)  # Set the Web Thickness to 18 mm for Layer 2
+    mc.set_array_variable("PoleArc_Array", 1, 105)  # Set the Pole Arc to 105 ED for Layer 2
```

Comment:
```suggestion
    mc.set_array_variable("WebThickness_Array", 1, 18)  # Set the web thickness to 18 mm for Layer 2
    mc.set_array_variable("PoleArc_Array", 1, 105)  # Set the pole arc to 105 ED for Layer 2
```

Reply:
Will keep capitalisation (from the parameter name in the UI, but cannot use bold for in-code comments)